src/components/blog-item.tsx

```diff
@@ -0,0 +1,71 @@
+import { styled, Typography } from '@mui/material'
+import Image from 'next/image'
+import Link from 'next/link'
+
+import blogStyles from '../styles/blog.module.css'
+
+interface blogItemProps {
+  image: string
+  title?: string
+  summary?: string
+  datePublished?: string
+  author?: string
+}
+
+const BlogItemContainer = styled('div')({
+  width: '48%',
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  margin: '20px 0',
+  '@media(max-width: 785px)': {
+    width: '68%',
+    flexWrap: 'wrap',
+  },
+})
+
+const BlogItemBody = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  margin: '0 20px',
+  '@media(max-width: 785px)': {
```

Comment:
The MUI theme actually comes with [preset breakpoints](https://mui.com/customization/breakpoints/). There's a bunch of ways to use them, [especially convenient](https://mui.com/system/basics/#responsive-values) with `sx prop` syntax. Don't worry about it for now though, I'll make a new issue for this later